docker-compose.yml

```diff
@@ -2,8 +2,14 @@ version: '3'
 services:
   rgit:
     image: ghcr.io/w4/rgit:main
+    command:
+      - "[::]:8000"
+      - /git
+      - -d /tmp/rgit-cache.db
     volumes:
-      - /path/to/my-repos:/git
+      - /volume/git:/git
     ports:
-      - 3333:8000
+      - 8000:8000
```

Comment:
Neither

Reply:
Ports shouldn't be exposed at all in a docker environment--but this is how people use docker, and is the way that the original `docker-composer.yml` file was to begin with. Yet again, it's going to depend on the end-users environment, so using `3333:8000` is tentatively identical to `8000:8000`.

`3333` interferes with NCID servers on the host machine, and `8000` interferes with some database servers. There's effectively nothing you can set it to that won't interfere with _something_, so again, I really don't understand the gripe here. They're both non-privileged ports.